appendices.adoc

```diff
@@ -26,32 +26,10 @@
 ===== CXL Protocols
 ===== PCIe Protocols
 ===== Graphics Protocols
-=== Recommended and Conditionally Required ACPI Tables
-==== I/O Topology
-==== TPM
-==== Platform Error Interfaces
-==== NUMA
-==== PCC
-==== Platform Debug Trigger
-==== NVDIMM
-==== Graphics Resource Table
-==== IPMI
-==== CXL
-==== iSCSI and NVMe-oF
-==== DBG2
-=== Recommended and Conditionally Required ACPI Methods
-==== CPU Performance Control
-==== CPU and System Idle Control
-==== NUMA
-==== IPMI
-==== Device Configuration and Control
-==== Resources
-==== CXL
-==== Time and Alarm
-==== PCI and PCIe
 === CXL Requirements
 ==== CXL Host Bridge
 ==== CXL Root Device
 ===== ACPI Device Object for CXL Root Device
 ==== NUMA
-
+=== Firmware Implementation Guidance
+==== ACPI Implementation Guidance
```

Comment:
What is this document? Sorry, I might have missed discussions around this since I cannot attend SEE meetings... But will this doc need to be ratified?

Reply:
What do you mean by 'this document'? The whole thing or this portion? The Guidance sections would be marked non-normative but provide helpful instructions for people implementing BRS.

Reply:
Ahh OK. I thought Implementation Guidance would be a separate document.